elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/example/Country.java

```diff
@@ -52,4 +55,33 @@ public String getName() {
     public void setName(final String name) {
         this.name = name;
     }
+
+    @Override
+    public boolean equals(final Object other) {
```

Comment:
Instead of generating these, can we just use lombok?  We should probably do that for all the schema, dimension, metric classes as well.

Reply:
Done for this class.

For others such as dimension and metric, looks like lombok is not happy with inheritance.